core/src/main/java/com/amplifyframework/predictions/models/IdentifiedText.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+import androidx.annotation.NonNull;
+
+import java.util.Objects;
+
+/**
+ * Represents the graphical text that is identified
+ * from inside an image.
+ */
+public final class IdentifiedText extends ImageFeature<String> {
+
+    private final int page;
+
+    private IdentifiedText(Builder builder) {
+        super(builder);
+        this.page = builder.getPage();
+    }
+
+    @Override
+    @NonNull
+    public String getTypeAlias() {
+        return FeatureType.IDENTIFIED_TEXT.getAlias();
+    }
+
+    /**
+     * Gets the identified text. This is the same
+     * text as the associated feature.
+     * @return the identified text
+     */
+    @NonNull
+    public String getText() {
+        return getValue();
+    }
+
+    /**
+     * Gets the page value.
```

Comment:
Like, page of a broader document? Or...? Huh.

Reply:
Currently there as a placeholder for platform parity. I will update the javadoc once I make a PR that actually uses this model